akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala

```diff
@@ -21,16 +21,18 @@ class TimeoutsSpec extends AkkaSpec {
 
     "pass through elements unmodified" in assertAllStagesStopped {
       Await.result(
-        Source(1 to 100).via(Timeouts.initalTimeout(2.seconds)).grouped(200).runWith(Sink.head),
+        Source(1 to 100).initialTimeout(2.seconds).grouped(200).runWith(Sink.head),
         3.seconds) should ===(1 to 100)
     }
 
     "pass through error unmodified" in assertAllStagesStopped {
       a[TE] shouldBe thrownBy {
         Await.result(
-          Source(1 to 100).concat(Source.failed(TE("test")))
-            .via(Timeouts.initalTimeout(2.seconds))
-            .grouped(200).runWith(Sink.head),
+          Source(1 to 100).
+            concat(Source.failed(TE("test"))).
+            initialTimeout(2.seconds).
+            grouped(200).
```

Comment:
Let's try no mixing different styles of "where to put the dot" ;) (most of this file is `\n.thing()\n.thing()`)